src/hooks/useUnitSuffix.tsx

```diff
@@ -0,0 +1,34 @@
+import BigNumber from "bignumber.js";
+
+const units = {
+  3: "K",
+  6: "M",
+  9: "B",
+  12: "T",
+};
+
+export enum UnitSuffixReturnValue {
+  SUFFIX = "suffix",
+  VALUE = "value",
+}
+
+export function useUnitSuffix(
+  value: string,
+  returnValue?: UnitSuffixReturnValue
+): string | BigNumber {
```

Comment:
hmm i rather you return an object that consists of `suffix` and `value`, then use object destructure to get what you want